.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
+    hooks:
+    -   id: flake8
+-   repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: v1.5
+    hooks:
+    -   id: autopep8
```

Comment:
I think we need to pass additional args to this hook so it produces the same results as running `tox`. See https://github.com/RockefellerArchiveCenter/pisces/blob/master/.pre-commit-config.yaml